mycroft/util/parse.py

```diff
@@ -216,7 +225,7 @@ def extract_duration(text, lang=None):
                     be None if no duration is found. The text returned
                     will have whitespace stripped from the ends.
     """
-    lang_code = get_primary_lang_code(lang)
+    lang_code = get_primary_lang_code(lang or get_default_lang())
     return lingua_franca.parse.extract_duration(text, lang_code)
```

Comment:
No need to pass a primary lang code here. LF will `extract_duration(lang='en-AU')` and that may even be desirable, when the number scales don't match!